pkg/cmd/server/api/v1/types.go

```diff
@@ -881,3 +888,12 @@ type LDAPQuery struct {
 	// Filter is a valid LDAP search filter that retrieves all relevant entries from the LDAP server with the base DN
 	Filter string `json:"filter"`
 }
+
+type AdmissionConfig struct {
+	// PluginConfig allows specifying a configuration file per admission control plugin
+	PluginConfig map[string]string `json:"pluginConfig"`
```

Comment:
> Having both master-config.yaml flags to turn on some admission plugins and then having a separate set of overrides seems pretty ugly, but I'm assuming that's the alternative you're considering?

@deads2k it sounded like you were saying have the admission plugins in the default set, so the cluster admin wouldn't have to muck with that, but they *would* have to specify configuration for the plugins to turn on force pulls. Is that accurate?